### DISCUSSION TITLE
Does anyone know where the Coral USB Accelerator is in stock?

### DISCUSSION THREAD

**Discussion by julianrinaldi** [blakeblackshear/frigate#546] [Ask A Question]
I have one and it works great with frigate. However I have 10 cameras running it on an RPi 4+, but if I add an 11th, then one of the streams gets messed up and shows artifacting/green blocks/etc. So I'd like to set up another instance. I can't find a Coral USB Accelerator in stock anywhere now though. Anyone know where to get one?

**Comment by blakeblackshear** [comment]
What resolutions/fps are you running on those 10 cameras?

**Reply by julianrinaldi** [reply]
1 x 960x540 2FPS
1 x 800x600 2FPS
8 x 640x360 2FPS

**Reply by julianrinaldi** [reply]
The issue does seem to be a bit random though. I've now been able to add 12 cameras without issue. But sometimes on restarting Frigate the image will be corrupted again.

**Reply by blakeblackshear** [reply]
I doubt the Coral is your limiting factor. It is almost certainly the USB3 bus or CPU of the Pi. Artifacts and green blocks would not be caused by the Coral maxing out.

**Reply by julianrinaldi** [reply]
Yes, I figured. I wanted to set up another instance of Frigate on another Rapsberry Pi 4.

**Reply by Trashmee** [reply]
I'm having the same issue with camera #4 - the only ReoLink in my setup. I've tried to mitigate by moving that from RTSP to RTMP. 
is your 11th camera different from the others? Funny thing is the camera keeps working fine for daysa & then there is a lot of smearing one day after restarts. 
I think its a ReoLink / ffmpeg related issue in my scenario.
If your camera 11 is the same model than all others - this post is not going to help you, sorry. Availability might be country dependent.

**Reply by alanpilz** [reply]
I have the same green screen problem.. I've been holding out adding too many cameras until I got my hands on a coral. 

It arrived today, and "appears to be working" detected in the logs etc..

I have 7 Dahua Cameras connected to a NVR, each running two streams, one for High Res playback (1920x1080), and one low res for detection. (640x480) with H.264 compression.

I had the green screen issue before I installed the Coral, it seems as soon as I add the last camera.. poof its green. I restarted HA, and a different camera was green... so it strongly suggests its not the actual feed or config. I was hoping the Coral would have helped..

Running on a Rasp Pi4 4Mb with a 480Gb M2 SSD (Argon One M2 Case) with a recently added Coral attached to USB3.0

**Reply by blakeblackshear** [reply]
@alanpilz are you using hwaccel? You are probably exceeding the available GPU memory on the Pi and will need to reduce the number of cameras using hwaccel args for ffmpeg. The Coral only does object detection. It does not help with decoding the video.

Green screen means ffmpeg is not sending any frames to frigate for processing.

**Reply by alanpilz** [reply]
Yes I am.. using the following for each camera.

`ffmpeg:
  hwaccel_args:
    - -c:v
    - h264_v4l2m2m`

Should I disable the acceleration settings?
To be honest the Pi is very slow just now.. Can't even open my configs.

![image](https://user-images.githubusercontent.com/85150538/125072018-9dd7c780-e0ec-11eb-9151-8303f8d59d6f.png)

**Reply by blakeblackshear** [reply]
@alanpilz can you open your own issue for support?

**Reply by alanpilz** [reply]
OK... Thanks... (my first time here...)

**Comment by HIT2022** [comment]
I reached out to Google Coral and they replied that the next batch of stock wasn't set to be shipped out to suppliers until around March, so I  think it will be difficult to find the usb version until then. Instead, I grabbed the PCIE M.2 model and its working fine (passed through from Proxmox host to Debian 'Supervised' Homeassistant install)

**Reply by walthowd** [reply]
Some more info on the M2 board for others here:
https://community.home-assistant.io/t/coral-edge-tpu-a-e-key-m-2-on-mini-itx-mobo/270905

**Reply by ljmerza** [reply]
I ordered the PICE M.2 was well just now. For docker-compose you add the USB coral by `- /dev/bus/usb:/dev/bus/usb`. What would I need to do to add the PCIE M.2 version?

**Reply by savikko** [reply]
And if you do not have M.2 connector but just PCIe on your machine, you can order mini-PCIe Coral and this: https://www.amazon.com/Ableconn-PEX-MP117-Mini-PCI-Adapter/dp/B07JBCL1CJ/

My config looks like this:
```yaml
detectors:
  coral1:
    type: edgetpu
    device: pci
```

**Reply by ljmerza** [reply]
oh sorry i mean my docker volume to add the pcie device to the container

**Reply by DeadEnded** [reply]
You'll have to check the device to make sure, but for my m.2 socket (linked above by walthowd) you pass the device through like this:
```
/dev/apex_0:/dev/apex_0
```
Again, you will have to check your host `/dev` and see what is created, but that worked for me.
This was the A+E key m.2 socket.

Cheers!
DeadEnd

**Reply by jcbbas** [reply]
Little late to the party here, but I just picked up one through Okdo, came from EU to the Midwest in 3 business days. Pretty pleased! Still showing in stock at the moment: https://www.okdo.com/us/p/coral-usb-accelerator/

**Reply by srlowe** [reply]
@jcbbas Funny, the website is telling me they can't ship to the EU, I think they are UK based.

**Comment by blakeblackshear** [comment]
Coral USBs are currently in stock here: https://www.okdo.com/us/p/coral-usb-accelerator/ 

UK link: https://www.okdo.com/p/coral-usb-accelerator/

**Reply by blakeblackshear** [reply]
Gone again

**Reply by LordNex** [reply]
Still out

**Comment by butako** [comment]
Thought you would like to know that Farnell UK is showing they will have stock for delivery on 2nd May 2022.
https://uk.farnell.com/pi3g/g950-01456-01/usb-accelerator-raspberry-pi/dp/3583033

**Reply by levy08** [reply]
> Thought you would like to know that Farnell UK is showing they will have stock for delivery on 2nd May 2022. https://uk.farnell.com/pi3g/g950-01456-01/usb-accelerator-raspberry-pi/dp/3583033

And it seems Farnell's US unit (Newark) is showing they'll have stock in June. I placed an order and it said it would be shipping from the UK:

 https://www.newark.com/coral/g950-01456-01/usb-accelerator-raspberry-pi-rohs/dp/26AJ4884?ost=CORAL++G950-01456-01

**Reply by gcortes** [reply]
I ordered one from the US unit. It cost me $106 with taxes and shipping. The unit itself is $88. The price has gone up, which isn't surprising.

**Reply by zillur01** [reply]
> I ordered one from the US unit. It cost me $106 with taxes and shipping. The unit itself is $88. The price has gone up, which isn't surprising.

When will you receive it?

**Reply by gcortes** [reply]
IIRC, it ships in the middle of June so it will be awhile

**Reply by gadgetbazza** [reply]
Latest update on my Farnell order from mid March 22 is delivery is now expected June 2023!

**Comment by Shred99** [comment]
Has anyone else noticed that with both RS Components and Element 14 (Farnell) the date that the Corals are due to arrive keeps slipping?  With RS Components, every day, the due date slips by one more day.

I've been chasing this mirage since last November.  The inability of these suppliers to give an accurate date that they can supply the device is getting extremely frustrating.

Bring on a version of Frigate that supports the Intel NCS2, which might not be as fast, but at least we can buy them.

**Reply by LaurenceGough** [reply]
They just want to get your order, then the date keeps changing as you say. Honestly I don't think we're getting stock any time soon but I do hope we do. I asked Coral directly (well Google) and they said they have shipped some out about a week or two ago but I saw no change in stock at all, they must have just gone to enterprise customers.

The good news is they are looking to get Asus on board to help produce Coral products, hopefully this will help, we'll have to wait to see.

**Comment by digiltd** [comment]
I was doing a casual search for the device recently and came across this article https://buyzero.de/blogs/news/google-coral-accelerator-jetzt-im-aiy-maker-kit-erhaltlich and they had in stock the maker kit, Pi4, Camera, USB Microphone, 32GB SD, Power Adaptor, and the Coral USB Accelerator. You had to buy the kit, though they did have a bundle of the sticks available individually but they have since sold out. The Kit is still in stock.

I was very skeptical, expecting the usual "we will ship it when we get it" email but two days later (I ordered on a German public holiday day) the kit was dispatched, four days later it was in my grubby hands in the UK, still a little bit doubtful as i could have opened it and still be told "we will ship the Coral when we get it" but no need to worry, there it was in all its glory. 

What made the whole thing sound too good to be true was the kit featured a Pi 4 8GB and Coral USB, two items that are hard to find (the Coral is nearly impossible to source) plus all the extra bits. Though I just checked, they actually sent the 4GB even though the website still says 8GB, tbh I am not too fussed, but I will raise it with them.

~~Good luck, still in stock~~ [not any more](https://github.com/blakeblackshear/frigate/discussions/546#discussioncomment-3043744) 

https://buyzero.de/collections/google-aiy-coral/products/aiy-maker-kit-includes-raspberry-pi-coral-usb-accelerator

**Reply by LaurenceGough** [reply]
Nice one finding that, a bit overpriced I think especially when adding shipping on top but quite amazing find considering the low stock of both items. Shame you got a 4GB model although that's more than enough for Frigate and HA, I'm running it off 2GB no issues at all even with a 1GB RAM volume for Frigate.

**Reply by antonbo** [reply]
Thank you for the recommendation. I was trying almost a year to find Coral USB. Ordered. Got it today to Sweden. All according to the list. Raspberry Pi4 8GB included

**Reply by funkyslap** [reply]
Thank you! Ordered Today from Italy 👍

**Reply by nempnemp** [reply]
Thank you! Arrived in western USA 16 days after the order. Coral currently working with Frigate, and Pi 4 has the 8GB chip. They gave me a USA-style AC adaptor. Shipping was €57, brutal. Looks like it's sold out now, alas.

**Reply by Gerben321** [reply]
Damn missed this. Please let me know when one is again in stock!

**Reply by digiltd** [reply]
> Damn missed this. Please let me know when one is again in stock!

orrrr... add it as a bookmark and set yourself a reminder to check it every couple of days :)

**Comment by Shred99** [comment]
I just received an email to say that the Coral I ordered last November from Element14 (Australia) has shipped.  Wow.  I'll believe it when I see it.

Checking in on the Element14 web site, the order code for the device that I ordered has disappeared and it has been replaced by a Google Coral with a new order code - and that one is on back-order until 1st of August.  I wonder if there has been slight model revision and the drought is finally breaking?

[EDIT: new order code on E14 site, still has the same manufacturer part number, so product is unchanged]

**Reply by Shred99** [reply]
> the Coral I ordered last November from Element14 (Australia) has shipped.

I received the Coral yesterday, so it looks like some of the backordered ones are finally starting to trickle through.  I've cancelled a couple of orders I had with RS Online, so if you've ordered through them since November 2021, you just move two places up the queue :- )

**Comment by LaurenceGough** [comment]
1110 Corals in stock on RS online in the UK just now. Hopefully they can send my one out soon ;). 

RIP to all the scalpers ;)

**Comment by gadgetbazza** [comment]
1108 ;)

Thanks BTW.  Off to cancel my 5 month order with Farnell.

**Reply by LaurenceGough** [reply]
No worries. I had a voucher to use with Farnell but you snooze you loose. 

I cannot believe how many there are after so long of a shortage, a bit like busses!

**Reply by gadgetbazza** [reply]
Yeah, I thought it must have been a typo when you put 1110!  Might hold onto my Farnell order until I get a despatch notice from RS.

**Reply by NickM-27** [reply]
Seems like ASUS taking over manufacturing may have started coming to fruition

**Reply by LaurenceGough** [reply]
I've noticed that ebay has been trickling away with a load in the last week or two, slowly coming down in prices. Someone somehow got a batch of them before others and tried to get lucky!

Looks like my stock alerts are working nicely ;).

RS online blocked my order which is nice, even though they assured me all was fine on the phone last time (I did ask them to confirm a few times!). I had to call again today to get it to move, I hope it's being dispatched today.

Typical I just 3d printed a stand for my Pi to hold it and the SSD nicely, I totally forgot about the USB accelerator!

**Reply by goodlucknow** [reply]
did anyone who placed an order with RS get a proper confirmation?
I received an automated order confirmation email, which says "We'll email you again to confirm prices and delivery dates".
but the order doesn't show up in my account history.
not sure if this is a normal part of the order process.

**Reply by ediguidedog** [reply]
> did anyone who placed an order with RS get a proper confirmation? I received an automated order confirmation email, which says "We'll email you again to confirm prices and delivery dates". but the order doesn't show up in my account history. not sure if this is a normal part of the order process.

I ordered a few weeks ago.  Like you I received a confirmation right away then an email a day later with estimateed delivery of mid August. A second email arrived a day or so later saying it would be this week but looking at my order on the  system it's early August again.  As long as it's soon I don't mind.  Basically once you get the second email you should be able to at least see the order.

**Reply by LaurenceGough** [reply]
Nah, nothing, I contacted them three times today as I pre-ordered this a while back and had a bit of an issue with the card used. I was told it would be dispatched today as it states on their website - free next day delivery.

A great shame if it doesn't arrive tomorrow after waiting so long it'd be nice!

This is my third order of the Coral now, at first when I ordered one at the end of last year and they put the date forward twice I cancelled it in frustration, now the price has gone up and I'm still getting it just as late haha.. typical.

Better than the Pi hut though, they're taking the pee with their price increase they're selling it for £78.50 now not even free delivery.

**Reply by goodlucknow** [reply]
Thanks for the info folks. So nothing in the order history until the second email, well hopefully my order gets processed before the UK stock runs out!

**Reply by Shred99** [reply]
Both Farnell/Element14 and RS Online have been all over the place with ETAs on these devices for more than six months now.  I wouldn't cancel your order with the opposition until you actually have your Coral in your hot little hand.

**Reply by gadgetbazza** [reply]
Something sounds a bit odd.  I got an email shortly after I completed my online order because of card difficulties apparently, even though they were 3d authenticated by the bank online...

I then phoned customer services as indicated on the email and they tried 2 other cards that wouldn't go through either.  They couldn't work out why.  They ended up making a fresh order and it went through without issue, thus far I have only had the order acknowledgment email.

I can however see the phone based order in my account online this morning and.... it's marked as despatched in the order summary, but in the tracking screen, still not marked as dispatched...  Who knows!

**Reply by goodlucknow** [reply]
Its strange alright. I've had no follow up email, but the order now shows in my account as despatched, quantity ordered: 1, quantity despatched: 0!!!
The stock levels also went from >2000 to out of stock last night, some ~600 this morning and now >1000.

**Reply by ediguidedog** [reply]
> Its strange alright. I've had no follow up email, but the order now shows in my account as despatched, quantity ordered: 1, quantity despatched: 0!!! The stock levels also went from >2000 to out of stock last night, some ~600 this morning and now >1000.

You should see a line below on the order marked "order in progress" and an estimated dispatch date. Mine is showing as the 27th July now

As an aside I've had a bit of a back and forward with support and despite me sending them a screenshot showing 2000 they said they had no idea why there was 2000 stock showing yesterday as they said there were none and sent me a screenshot (which showed none). This morning I emailed them again when it was at a stock of 51 and its risen since to 1570 available as I speak. So far no further response from support. So far they have been good at getting back to me so I'll let you all know when they do.

**Reply by ediguidedog** [reply]
> So far they have been good at getting back to me so I'll let you all know when they do.

And they just have. They confirmed the stock exists and that one will be with me "in the next few days". It still shows as "Order in progress" but things are looking hopeful

**Reply by goodlucknow** [reply]
That's good. I spoke with support at the local outlet and they said it was in stock also(via UK).
The site looks a bit different for me, could be a regional thing. However I did get the follow up email, and they've charged my card now.
Estimated despatch today but won't hold my breath till I see a shipping number!

**Reply by gadgetbazza** [reply]
Got my dispatch notification around 5-6pm so fingers crossed.

**Reply by LaurenceGough** [reply]
Mine is coming tomorrow now via DHL (y)  it's been fun guys, I hope you all manage to get one too. The stock number and tales told here do seem a bit odd, perhaps they are stating they have more than they do, who knows. I think I may try the Coral full power mode at first but I think the half power mode will be more than plenty with my current setup. My Pi probably couldn't take the full power mode anyway without a powered USB hub...

**Reply by ediguidedog** [reply]
>  I think I may try the Coral full power mode at first but I think the half power mode will be more than plenty with my current setup. My Pi probably couldn't take the full power mode anyway without a powered USB hub...

I've read about the two power modes but I'm not aware of any settings in frigate that can influence that.  Which mode does frigate use by default and how can it be changed.

**Reply by NickM-27** [reply]
Frigate runs the coral at full power, there is not a way to change that unless you change the source and build yourself (AFAIK).

Either way, USB powered hub is highly recommended to avoid any weird behavior

**Reply by LaurenceGough** [reply]
> 

Ah crap, I was hoping to avoid more cables and power supplies, RIP. I'd have used the PoE hat for my Pi if it didn't drain so much power and run so hot by itself. The one I had delivered was DoA anyway! 

This is where I found out about it https://coral.ai/docs/accelerator/get-started/#1-install-the-edge-tpu-runtime

I know absolutely nothing about it however and my Linux/acceleration methods knowledge is pretty much non existent.

**Reply by ediguidedog** [reply]
> Frigate runs the coral at full power, there is not a way to change that unless you change the source and build yourself (AFAIK).
> 
> Either way, USB powered hub is highly recommended to avoid any weird behavior

Perfect thanks

**Reply by NickM-27** [reply]
@LaurenceGough That is something that Frigate does at time of build https://github.com/blakeblackshear/frigate/blob/release-0.11.0/docker/Dockerfile#L81

**Reply by LaurenceGough** [reply]
Some ideas here if you are using a Pi (of course the powered USB hub will work too).
https://www.reddit.com/r/homeassistant/comments/kvoavr/raspberry_pi_4_undervolting/

When I calculated it if you are using a low powered eco SSD and nothing else in the USB ports and a quality 5v power supply it should be ok with the Pi 4 but these calculations were so many months ago now, I think last year even... SSDs seem to vary quite a bit in power consumption as noted on a few Pi support sites.


The Pi 4 itself uses a lot less power on the latest firmware, running it cool also helps reduce power usage. There is also the option to lower the clock/voltage on the Pi if you can get away with it but if I recall the bigger issue was with the USB limit on the Pi itself, there are many options to work around it though. As Nick said just use a powered USB hub if you want a simple life :D.

I might be the only one here using a Pi though :D 

Google / Coral suggest running it at standard speed for environments above 25c. That'll be interesting with the current hot temps here. I'm hoping that rating is with 24/7 stressful use in mind and not the occasional image detection that Frigate runs.

**Reply by LaurenceGough** [reply]
> @LaurenceGough That is something that Frigate does at time of build https://github.com/blakeblackshear/frigate/blob/release-0.11.0/docker/Dockerfile#L81

Thanks Nick, super knowledgeable and helpful as always. It's much appreciated your help here.

**Comment by LaurenceGough** [comment]
2115 In stock now @ RS Online :o Even more showing!

**Reply by kusold** [reply]
I see it on the [UK site](https://uk.rs-online.com/web/p/development-tool-accessories/2017821), but not the US. UK checkout requires you to select a UK county, so I'm unsure if there is a way to ship to the US.

Have any US-based people succeeded in ordering this?

**Reply by LaurenceGough** [reply]
If you have no luck let me know and I could look into ordering one and shipping it to you, probs cheaper than those services that offer that. Postage will probably be a fair bit though...

It might be worth waiting a bit to see if other retailers get them in stock, I believe Pi hut does international shipping as others do probably. Okdo might be another one worth checking out.

I must admit I have fine tuned my Pi so much now that the CPU based one is just fine unless there is more than one person being tracked.

**Comment by kirilldem** [comment]
Shipped, got a ups tracking number, thanks for the heads up @LaurenceGough

**Comment by TD22057** [comment]
I went ahead and ordered a USB model from Farnell's US website for $107.  It had one line that said it would be shipped in UK stock from 2-4 days and another line that said available for back order with est delivery in Sept.

**Reply by TD22057** [reply]
FYI - it didn't ship in 2-4 days but they did update the expected shipping date to Aug 19th.  Total with tax and shipping was $130 which is kind of annoying but it's still cheaper than ebay.  For those of you in the US that might be a good option.

**Comment by gadgetbazza** [comment]
Tada...  Rs Stock is genuine...
![IMG_1317](https://user-images.githubusercontent.com/99490699/180492499-ff0b6df0-12ba-47c0-b5c3-a15a5a3b1cf7.jpeg)

**Reply by LaurenceGough** [reply]
Get plugging in :). Mine is already in and running hehe. Super fast and no power issues yet (touch wood). That's with my still slightly over clocked Pi haha.

Simple config change restarted frigate and boom. I used the opportunity to upgrade to the latest Beta version too. All working great. 

It runs pretty hot even at idle which is disappointing but other than that all good.

**Reply by gadgetbazza** [reply]
Great stuff, my Frigate is running in a container on a QNAP NAS, so just need to find the docker instructions to pass the USB through first and then onto Frigate config.  My inference from two i7 cores is currently 43.0 - 43.4. Will be interesting to see how much it drops.  Poor CPU is busy most of the time checking 18 cameras with 5 to add still!

**Comment by ml-tx** [comment]
685 in stock at berrybase:
https://www.berrybase.de/google-coral-usb-accelerator-f-252-r-raspberry-pi

**Comment by SgtBatten** [comment]
Still waiting on my coral from mouser Australia since September wtf.

**Reply by SgtBatten** [reply]
Got it last month finally.

**Comment by jakegibson** [comment]
https://buyzero.de/en/products/aiy-maker-kit-includes-raspberry-pi-coral-usb-accelerator
Back in stock if you needed a PI as well. Shipping to the US for two kits was ~65

**Comment by operator207** [comment]
If you want to buy one from Amazon, they are down to[ $139 (3rd party)](https://www.amazon.com/gp/product/B07R53D12W)
If you want it shipped from Amazon (so you can return it if it doesn't work out,[ right now, it's still $139](https://www.amazon.com/Google-G950-01456-01-Coral-USB-Accelerator/dp/B07S214S5Y).
I bought one the other day for $158 as my m.2 one from Mouser (US) was still on back order. I'll leave it like that, cause I might be able to cram it into a system I have laying around that has an open m.2 slot later on.

**Comment by dnestico** [comment]
Probably a dumb question but just got my coral TPU USB and just wanna make sure how to use it.

Do I just add the line of code from the config example to my frigate.yml then plug in the USB Coral and it will just recognize it? Or do I need any other addon? Do I need to turn the RPI4 off first before doing this?

Just want to make sure before I add my Coral USB.

**Reply by NickM-27** [reply]
It should just be able to plug it in and run it. If you have an Rpi then a powered USB hub is recommended.

**Reply by dnestico** [reply]
Thanks! And damn I did not buy a powered USB hub, would it still work without one for now, then I can get one later on?

**Reply by NickM-27** [reply]
You can try it, but if it doesn't work or intermittently works then that would be the recommendation

**Reply by dnestico** [reply]
Any USB power hubs you recommend?

Links would be nice, thanks.

**Reply by NickM-27** [reply]
There's many that will work great, I use https://www.bestbuy.com/site/j5create-7-port-usb-3-0-hub-black/4561575.p?skuId=4561575 with my test Pi

**Comment by TD22057** [comment]
FYI I signed up for notifications from https://thepihut.com and it pinged a few days ago.  Got a USB Coral for 66£ + 20£ shipping.  DHL tracking number was sent out later that day.  They're out of stock again but it appears stock is starting to show up.

**Comment by gcortes** [comment]
I just got my shipping notice from Newark that my Coral is on it's way. I ordered it in April so only six months. Newark is the US arm of a British company, the name of which I've forgotten.

**Comment by sykick** [comment]
Currently showing in stock here: https://thepihut.com/products/coral-usb-accelerator

**Comment by rayzorben** [comment]
All sold out everywhere I look, or $190 USD+ :(

**Comment by heebo1974** [comment]
If you need Coral in Europe:
https://www.welectron.com/Google-Coral-USB-Accelerator_1

**Reply by chollllohc** [reply]
Just ordered one from there and being shipped to US... hopefully! Thanks for the link.

**Reply by loidolt** [reply]
Also ordered one to the US, appreciate the heads up!

**Comment by denveronly** [comment]
anyone know where in europe can i get coral now?
ebay pisses me off with 170-200$

**Comment by tpjanssen** [comment]
You can find them [here](https://www.antratek.nl/coral-usb-accelerator) . Pretty expensive, but I guess that is same everywhere.

**Comment by ghost** [comment]
Botland in Poland have them and ship to anywhere in Europe, 
[Link](https://botland.store/google-coral/16911-google-coral-usb-accelerator-edge-tpu-ml-arm-cortex-m0-193575021935.html)

**Comment by ghost** [comment]


**Reply by Iceman248** [reply]
I gave up on the Coral after trying out the new OpenVino detector option. It has worked great for me.

**Comment by kyle4269** [comment]
> ordered one from mouser. the package got stolen enroute in texas. back to energy-burning GPUs it is unless anyone wants to pool together 550 people and order em in bulk from Google :moneybag: 

I have an extra if you're interested. Figured I'd offer here before I put it on ebay

**Reply by kyle4269** [reply]
Ahh sorry. Bought this for $110. Just looking for what I paid

**Comment by denveronly** [comment]
Brought at botland poland
Im pretty happy with it
Inference speed drop from 100ms to 8ms
Cpu load 75% drop to 30 (i5-8500)

7 cameras
Really is great
